src/frontends/paddle/src/CMakeLists.txt

```diff
@@ -5,6 +5,5 @@
 ov_add_frontend(NAME paddle
                 LINKABLE_FRONTEND
                 PROTOBUF_REQUIRED
-                PROTOBUF_LITE
```

Comment:
why have you remove it?

Reply:
Because the CI and my local env have the following error when I build:

```
Error Message: undefined reference to google::protobuf::Message::CopyWithSizeCheck
```